python/lsst/summit/utils/peekExposure.py

```diff
@@ -968,10 +970,13 @@ def runPeek(
             # donut mode.
             if donutDiameter > self.config.donutThreshold:
                 mode = "donut"
-            elif isDispersedExp(exposure):
-                mode = "spec"
             else:
-                mode = "photo"
+                if exposure.getInfo().getVisitInfo().instrumentLabel == "LATISS":
+                    # only LATISS images *can* be dispersed, and isDispersedExp
+                    # only works cleanly for LATISS
+                    mode = "spec" if isDispersedExp(exposure) else "photo"
+                else:
+                    mode = "photo"
```

Comment:
FWIW, and now we're probably just in the land of personal preference, but I'd write that as:
```
mode = "photo"
if exposure.getInfo().getVisitInfo().instrumentLabel == "LATISS":
    # only LATISS images *can* be dispersed, and isDispersedExp
    # only works cleanly for LATISS
    mode = "spec" if isDispersedExp(exposure)
```
because otherwise you end up with a variable that's defined inside if/else blocks which could end up, if the logic changed in the future, undefined, and this way you'll always have the fallback. Definitely in the noise though 🙂

Reply:
Oh, except that's a supplied arg... anyway, whatever, all is good, this looks great 🙂 👍